cybox/common/vocabs.py

```diff
@@ -6,13 +6,58 @@
 from cybox.common import PatternFieldGroup
 from cybox.utils import normalize_to_xml, denormalize_from_xml
 
-INVALID_XSI_TYPE = "bad_xsi_type"
+
+class VocabField(cybox.TypedField):
```

Comment:
I think I have a decent idea how this class works, but it could use a bit better documentation? Is the idea that any VocabField is forced to be of type VocabString, but you can use the `type_` parameter to specify which vocab to use. I almost thing we should just add a separately-named arg (though that also has its downsides).